proto/regen/ecocredit/v1/events.proto

```diff
@@ -159,3 +162,15 @@ message EventAddCreditType {
   // abbreviation is the abbreviation of the credit type.
   string abbreviation = 1;
 }
+
+// EventBridge is emitted emitted when credits are cancelled.
+message EventBridge {
+  // bridge_target is the target chain.
+  string bridge_target = 1;
+
+  // bridge_recipient is the recipient address.
+  string bridge_recipient = 2;
+
+  //  bridge_contract is the contract address.
+  string bridge_contract = 3;
+}
```

Comment:
Although included in the issue, I think we should shorten these names and exclude the `bridge_` prefix.

cc @clevinson is that ok with you?

```suggestion
// EventBridge is emitted emitted when credits are bridged to another chain.
message EventBridge {
  // target is the target chain.
  string target = 1;

  // recipient is the recipient address.
  string recipient = 2;

  // contract is the contract address.
  string contract = 3;
}
```